wireapp/wire-android
# feat: epic send connection request (AR-1415)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1415" title="AR-1415" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-1415</a>  Send connection request
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

This work it's the final PR containing all the previous work separated into smaller PRs, enabling the feature of "Send a connection request"

- #576 
- #579 

### Dependencies (Optional)

Needs releases with: https://github.com/wireapp/kalium/pull/508


### Testing

#### Test Coverage (Optional)


#### How to Test

You can check out this branch and send a connection request to another user, you should be able to check on the other end (user who receives the invitation) the pending connection request received.

### Notes (Optional)

Some things were left out, since will be addressed in upcoming tickets, for example showing in the list of conversations/search results the pending status of connection

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
